.github/workflows/cifuzz.yml

```diff
@@ -1,18 +1,17 @@
 name: CIFuzz
 on:
-  # comments out triggerin on PR for now
-  # pull_request:
-  #   paths:
-  #    - '**.c'
-  #    - '**.cc'
-  #    - '**.cpp'
-  #    - '**.cxx'
-  #    - '**.h'
-  # Only trigger by post merge for now.
+  pull_request:
+    branches: '**'
+    paths:
+      - 'icu4c/**'
```

Comment:
Is it really desireable to require this to be run also for changes that don't touch any source code?